src/service/action-impl.js

```diff
@@ -733,22 +735,39 @@ export class ActionService {
   }
 }
 
-
 /**
  * Returns `true` if the given action invocation is whitelisted in the given
- * whitelist.
+ * whitelist. Default actions' alias, 'activate', are automatically
+ * whitelisted if their corresponding registered alias is whitelisted.
  * @param {!ActionInvocation} invocation
  * @param {!Array<{tagOrTarget: string, method: string}>} whitelist
  * @return {boolean}
  * @private
  */
 function isActionWhitelisted_(invocation, whitelist) {
+  const {method, node, tagOrTarget} = invocation;
+  const tagOrTargetRegex = caseInsensitiveMatch(tagOrTarget);
+  let invocationMethodRegex = caseInsensitiveMatch(method);
+  const defaultActionAlias = node.getDefaultActionAlias();
+  if (method == DEFAULT_METHOD && (node
+      && defaultActionAlias)) {
```

Comment:
This should ideally be two-way, right? E.g. for an AMP component that has a default action "open", whitelisting either "activate" or "open" should work.

Reply:
Fair enough. Can we make this conditional a bit more readable though?

Reply:
done. Let me know if it's still not clear or if I can improve it.